src/main/kotlin/com/msg/gcms/domain/attendance/util/impl/AttendanceConverterImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.msg.gcms.domain.attendance.util.impl
+
+import com.msg.gcms.domain.attendance.domain.entity.Attendance
+import com.msg.gcms.domain.attendance.domain.entity.Schedule
+import com.msg.gcms.domain.attendance.domain.enums.AttendanceStatus
+import com.msg.gcms.domain.attendance.util.AttendanceConverter
+import com.msg.gcms.domain.user.domain.entity.User
+import org.springframework.stereotype.Component
+
+@Component
+class AttendanceConverterImpl : AttendanceConverter {
+    override fun toEntity(
+        attendanceStatus: AttendanceStatus,
+        user: User, schedule: Schedule
```

Comment:
```suggestion
        user: User,
        schedule: Schedule
```
요기 혹시 들여쓰기 안되어있는거 의도인건가요?